packages/docs/static/themes/medicare-theme.css

```diff
@@ -132,354 +132,354 @@
+--steplist-step__border-color: var(--color-border);
+--steplist-step__border-color--default: var(--color-muted);
+--steplist-step__background-color--completed: var(--color-base);
+--steplist-step__color: var(--color-muted);
+--steplist-step__color--current: var(--color-white);
+--steplist-step__color--completed: var(--color-base);
+--review__border-color: var(--color-border);
+--spinner__background-color: var(--color-background);
+--spinner__color: var(--color-base);
+--spinner__color--inverse: var(--color-white);
+--spinner__background-color--inverse: var(--color-background-inverse);
+--table__border-color: var(--color-black);
+--table__background-color--striped: var(--color-gray-lightest);
+--table__padding: var(--spacer-2);
+--table-header__background-color: var(--color-gray-lightest);
+--tabs__border-color: var(--color-border);
+--tabs__background-color: var(--color-background);
+--tabs__background-color--selected: var(--color-background);
+--tabs__background-color--hover: var(--color-background);
+--tabs__background-color--disabled: var(--color-gray-lighter);
+--tabs__border-color--selected: var(--color-primary);
+--tabs__border-color--disabled: var(--color-gray-lighter);
+--tabs__color: var(--color-base);
+--tabs__color--hover: var(--color-primary);
+--tabs__color--selected: var(--color-primary);
+--tabs-panel__background-color: var(--color-background);
+--tabs__color--active: var(--color-primary-darker);
+--tabs__color--disabled: var(--color-gray-darker);
+--text-input__background-color--disabled: var(--color-gray-lighter);
 --text-input__border-width: 2px;
 --text-input__border-width--disabled: 1px;
 --text-input__border-width--error: 3px;
---text-input__border-color: #808080;
---text-input__border-color--disabled: #404040;
---text-input__border-color--error: #b20000;
---text-input__border-color--error--inverse: #e8b3b3;
---text-input__border-color--inverse: #808080;
---text-input__border-radius: 3px;
---text-input__color: #404040;
---text-input__divider-color: #a6a6a6;
---text-input__padding: 16px;
+--text-input__border-color: var(--color-gray);
+--text-input__border-color--disabled: var(--color-gray-darker);
+--text-input__border-color--error: var(--color-error);
+--text-input__border-color--error--inverse: var(--color-error-lighter);
+--text-input__border-color--inverse: var(--color-gray);
+--text-input__border-radius: var(--radius-default);
+--text-input__color: var(--color-gray-darker);
+--text-input__divider-color: var(--color-gray-light);
+--text-input__padding: var(--spacer-2);
 --text-input__line-height: 1.2999999523162842;
---text-input-currency-icon__line-height: 1;
+--text-input-currency-icon__line-height: var(--font-line-height-reset);
 --text-input-currency-icon__spacing: 32px;
---tooltip__background-color: #ffffff;
---tooltip__border-color: #404040;
+--tooltip__background-color: var(--color-background);
+--tooltip__border-color: var(--color-gray-darker);
 --tooltip__border-color--active: #146a5d40;
```

Comment:
odd hex

Reply:
You know, one idea for syncing from figma to tokens would be to take an incoming 4-channel hex, chop off the alpha, and then see if it matches any of the existing variable's 3-channel hex colors and turn that into `rgba(var(--color), 0.8)`. That's a CSS value, though, and that's not what we expect to store in the JSON tokens. We'd need to store it in some new, composite type that references another color token and adds alpha.

Reply:
I think that's a future enhancement.